receiver/cloudfoundryreceiver/converter.go

```diff
@@ -4,22 +4,28 @@
 package cloudfoundryreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudfoundryreceiver"
 
 import (
+	"errors"
 	"time"
 
 	"code.cloudfoundry.org/go-loggregator/rpc/loggregator_v2"
 	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
 const (
-	attributeNamePrefix = "org.cloudfoundry."
+	attributeNamePrefix        = "org.cloudfoundry."
+	envelopeSourceTypeKey      = "source_type"
+	envelopeSourceTypeValueRTR = "RTR"
+	logLineRTRZipkinKey        = "b3"
+	logLineRTRW3CKey           = "traceparent"
+	traceW3CVersion            = "00"
```

Comment:
From what I can tell there are no current usages of any of these introduced consts. Do we need them yet, or can they be removed?

Reply:
Removed